foxglove_bridge_base/include/foxglove_bridge/common.hpp

```diff
@@ -145,4 +147,16 @@ struct ServiceResponse {
 
 using ServiceRequest = ServiceResponse;
 
+enum FetchAssetStatus {
```

Comment:
- Prefer `enum class` to avoid polluting the namespace
- Add `: uint8_t` to match the spec